### PR TITLE
Add sample script to grab stats over the REST API.

### PR DESCRIPTION
**Pull request by jaymzh**

### CODE REVIEW

File: utils/bmc_stats.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  '/api/sys/gpios',
+  '/api/sys/modbus_registers',
+  '/api/sys/psu_update',
+  '/api/sys/slotid',
+]
+
+pp = pprint.PrettyPrinter()
+
+def get_api(host, api):
+  print("=== %s ===" % api)
```
Comment (theopolis) [inline utils/bmc_stats.py]: Should we enforce more pep8-ish? And nit for space counts?

File: utils/bmc_stats.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
```
Comment (theopolis) [inline utils/bmc_stats.py]: Facebook copyright needed?

File: utils/bmc_stats.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  for api in apis:
+    get_api(host, api)
+
+def list_apis():
+  for api in apis:
+    print api
+  print 'all'
+
+
+parser = OptionParser()
```
Comment (theopolis) [inline utils/bmc_stats.py]: Perhaps drop this into a `if __name__ == '__main__'` incase anyone wanted to build on / import from these.

Comment (jaymzh) [inline utils/bmc_stats.py]: Happy to. I think you're giving this terrible example script far more credit than it's worth though. ;)

Comment (theopolis) [inline utils/bmc_stats.py]: Haha! But there's lots of value in having example scripts, so I'd like this to set an example-example!

**Comment by jmehta** [comment]
Hello @jaymzh and @theopolis, Any updates on this PR? If not I request we close it and open a new one against `helium` branch.

**Comment by jaymzh** [comment]
Sorry I dropped this. I can update the PR.

**Comment by jaymzh** [comment]
OK, @theopolis - pep8-ified it, 4-space-ified it, add the main check and added the copyright.

**Comment by jmehta** [comment]
Closing this one for now. @jaymzh, can you please open a new PR against the `helium` branch.

**Pull request closed without merging**